### PR TITLE
Complete ticket 10-get-api-reviews-queries

### PR DESCRIPTION
**Pull request by Nanobot1514**
Complete ticket 10. Handled 3 errors. Modified the fetchReviews function in models to accept queries

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -82,6 +82,74 @@ describe("app", () => {
...
+          });
+      });
+      it("responds with reviews in the correct order when given an ?order_by query", () => {
+        return request(app)
+          .get("/api/reviews?category=social deduction&sort_by=title&order=asc")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.reviews).toHaveLength(11);
+            expect(body.reviews).toBeSortedBy("title", { descending: false });
+          });
```
Comment (Jodiech) [inline __tests__/app.test.js:126]: You should add a 200 test for no articles that responds with an empty array of articles for example ?category=children's games.

File: models/reviews-models.js [hunk 2]
```diff
@@ -1,16 +1,51 @@
...
+    query += ` WHERE reviews.category = $1`;
+    queryParams.push(category);
+  }
+
+  if (sort_by && order) {
+    query += `
+  GROUP BY reviews.review_id 
+  ORDER BY ${sort_by} ${order};`;
+  }
+
```
Comment (Jodiech) [inline models/reviews-models.js:55]: Great to see you have built this query string dynamically 👍

**Review by Jodiech — CHANGES_REQUESTED** [review]
Great work! I have added a comment for you to add another 200 test (please see comments below) once you have done this I am happy for you to merge 😄
